distributed/http/static/css/status.css

```diff
@@ -6,45 +6,53 @@
 @media (min-width: 0px) {
   #status-fluid {
     grid-template-columns: 1fr 1fr;
-    grid-template-rows: 1fr 3fr 1fr;
+    grid-template-rows: 1fr 2fr 6fr 2fr;
   }
-  #status-history {
-    grid-column: 1;
+  #status-nbytes-cluster {
+    grid-column: 1 / span 2;
     grid-row: 1;
   }
+  #status-nbytes-workers {
+    grid-column: 1;
+    grid-row: 2;
+  }
   #status-processing {
     grid-column: 2;
-    grid-row: 1;
+    grid-row: 2;
   }
   #status-tasks {
     grid-column: 1 / span 2;
-    grid-row: 2;
+    grid-row: 3;
   }
   #status-progress {
     grid-column: 1 / span 2;
-    grid-row: 3;
+    grid-row: 4;
   }
 }
 
 @media (min-width: 992px) {
   #status-fluid {
     grid-template-columns: 1fr 3fr;
-    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 4fr 1fr 1fr 4fr;
```

Comment:
FIXME the new cluster-wide RAM usage widget gets completely invisible if the browser window is too short. Need to find a way to set a minimum absolute height.

Reply:
I am seeing this with the dashboard full screen on a 15" MacBook Pro. I expect this will affect a lot of users.

<img width="373" alt="image" src="https://user-images.githubusercontent.com/1610850/115251233-409e2c80-a122-11eb-9c38-1b6e918fb147.png">

Reply:
```suggestion
    grid-template-rows: 2fr 4fr 1fr 1fr 4fr;
```

Reply:
<img width="1432" alt="image" src="https://user-images.githubusercontent.com/1610850/115254559-5f51f280-a125-11eb-86bd-6236c12f25ae.png">